crypto/params.c

```diff
@@ -62,26 +63,173 @@ void OSSL_PARAM_set_all_unmodified(OSSL_PARAM *p)
             p++->return_size = OSSL_PARAM_UNMODIFIED;
 }
 
+/* Return non-zero if the signed number is negative */
+static int is_negative(const void *number, size_t s)
+{
+    const unsigned char *n = number;
+    DECLARE_IS_ENDIAN;
+
+    return 0x80 & (IS_BIG_ENDIAN ? *n : n[s - 1]);
```

Comment:
Nit: perhaps `n[0]` instead of `*n`, for code consistency on this line?

Reply:
Not a huge fan but I'll change it.